dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.service.description.3.2/bnd.bnd

```diff
@@ -34,4 +34,10 @@ Export-Package: \
   @${repo;org.apache.cxf:cxf-rt-rs-service-description;${cxfVersion}}!/!META-INF/*
 
 -buildpath: \
-  org.apache.cxf:cxf-rt-rs-service-description;version=${cxfVersion}
+	org.apache.cxf:cxf-rt-rs-service-description;version='${cxfVersion}',\
```

Comment:
Don't forget to add the   `org/apache/cxf=${bin}/org/apache/cxf` line to the include resources header, so that you make sure the overlay ends up in the bundle jar.